src/main/java/org/maxgamer/rs/core/Core.java

```diff
@@ -167,8 +172,7 @@ public synchronized static Cache getCache() {
                 cache.setRaw(255, IDX.LANDSCAPES, f.encode());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this call to "exit" or ensure it is really required. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://titan.maxgamer.org:9000/coding_rules#rule_key=squid%3AS1147)